### PR TITLE
Api_PUT_invalid

### PR DESCRIPTION
**Pull request by KytaihoraVolodymyr**

### CODE REVIEW

File: spec/api/api_client.rb [hunk 1]
```diff
@@ -9,6 +9,9 @@ def initialize
     @base_url = 'https://petstore.swagger.io/v2/user/'
   end
 
+  APP_JS = "application/json"
```
Comment (AndriiMish) [inline spec/api/api_client.rb]: duplicated line

File: spec/api/api_client.rb [hunk 2]
```diff
@@ -9,6 +9,9 @@ def initialize
...
+  DEFAULT_PAGE = "https://petstore.swagger.io/v2/user/"
```
Comment (AndriiMish) [inline spec/api/api_client.rb]: do we need it? where do we use this page?

File: spec/api/api_put_invalid_spec.rb [hunk 3]
```diff
@@ -0,0 +1,299 @@
+require_relative '../spec_helper'
+require_relative 'api_client.rb'
+require 'securerandom'
+
+RSpec.describe 'Create -> Login -> Get' do
+
+  app_cl = ApiClient.new
+  body = app_cl.generate_random_body
+
+  before(:all) { app_cl.login_as_default_user }
```
Comment (AndriiMish) [inline spec/api/api_put_invalid_spec.rb:10]: nice!

File: spec/api/api_put_invalid_spec.rb [hunk 4]
```diff
@@ -0,0 +1,299 @@
...
+            response = app_cl.update_user(body[:username], body)
+            expect(response.status).to eq(400)
+        end
+
+        it 'verifies user can update with numerical values' do
+            body[:firstName] = 123456
+            response = app_cl.update_user(body[:username], body)
+            expect(response.status).to eq(400)
+        end
+    end
```
Comment (AndriiMish) [inline spec/api/api_put_invalid_spec.rb:100]: consider using shared examples or hashes to reduce code duplication

Comment (KytaihoraVolodymyr) [reply spec/api/api_put_invalid_spec.rb:100]: working on optimization now

**Pull request merged**